src/land/imsfv3_scf2ioda.py

```diff
@@ -63,24 +69,26 @@ def _read(self):
             if iodavar == 'snowCoverFraction':
                 self.varAttrs[iodavar, iconv.OvalName()]['units'] = '1'
                 self.varAttrs[iodavar, iconv.OerrName()]['units'] = '1'
+                # should use float_missing_value
                 self.varAttrs[iodavar, iconv.OvalName()]['_FillValue'] = -999.
                 self.varAttrs[iodavar, iconv.OerrName()]['_FillValue'] = -999.
```

Comment:
```suggestion
                # should use float_missing_value
                self.varAttrs[iodavar, iconv.OvalName()]['_FillValue'] = float_missing_value
                self.varAttrs[iodavar, iconv.OerrName()]['_FillValue'] = float_missing_value
```

@YoulongXia-NOAA here is the code looking explictly for the `-999.` value or can I change it to `float_missing_value`... of course also need to modify the code to do the same change